packages/network-of-terms-reconciliation/test/server.test.ts

```diff
@@ -61,6 +61,25 @@ describe('Server', () => {
     expect(results.q3.result).toEqual([]); // No results.
   });
 
+  it('limits reconciliation API results', async () => {
+    const response = await reconciliationQuery(
+      'https://data.netwerkdigitaalerfgoed.nl/rkd/rkdartists/sparql',
+      {
+        q1: {
+          query: 'art',
+        },
+        q2: {
+          query: 'art',
+          limit: 1,
```

Comment:
Are there restrictions to the `limit`? For example: can I send `limit: -1` or `limit: "blah"`?

Reply:
Good point! Added JSON Schema-based validation of reconciliation requests.

Reply:
Nice!